diff --git a/pymediainfo/__init__.py b/pymediainfo/__init__.py
index 9c18679..1140c77 100644
--- a/pymediainfo/__init__.py
+++ b/pymediainfo/__init__.py
@@ -301,7 +301,7 @@ class MediaInfo:
     ) -> Tuple[Any, Any, str, Tuple[int, ...]]:
         os_is_nt = os.name in ("nt", "dos", "os2", "ce")
         if os_is_nt:
-            lib_type = ctypes.WinDLL  # type: ignore
+            lib_type = lambda f: ctypes.WinDLL(f, winmode=0)  # type: ignore
         else:
             lib_type = ctypes.CDLL
         if library_file is None:
